src/main/java/com/artipie/api/artifactory/GetUserSlice.java

```diff
@@ -0,0 +1,105 @@
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.api.artifactory;
+
+import com.artipie.Settings;
+import com.artipie.http.Response;
+import com.artipie.http.Slice;
+import com.artipie.http.async.AsyncResponse;
+import com.artipie.http.rq.RequestLineFrom;
+import com.artipie.http.rs.RsStatus;
+import com.artipie.http.rs.RsWithBody;
+import com.artipie.http.rs.RsWithStatus;
+import com.artipie.http.rs.StandardRs;
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.json.Json;
+import org.reactivestreams.Publisher;
+
+/**
+ * Artifactory `GET /api/security/users/{userName}` endpoint, returns user information.
+ * @since 0.10
+ */
+public final class GetUserSlice implements Slice {
+
+    /**
+     * Request line pattern to get username.
+     */
+    private static final Pattern PTRN = Pattern.compile("/api/security/users/(?<username>[^/.]+)");
+
+    /**
+     * Artipie settings.
+     */
+    private final Settings settings;
+
+    /**
+     * Ctor.
+     * @param settings Artipie setting
+     */
+    public GetUserSlice(final Settings settings) {
+        this.settings = settings;
+    }
+
+    @Override
+    public Response response(final String line, final Iterable<Map.Entry<String, String>> headers,
+        final Publisher<ByteBuffer> body) {
+        final Response res;
+        final Matcher matcher = GetUserSlice.PTRN.matcher(
+            new RequestLineFrom(line).uri().toString()
+        );
+        if (matcher.matches()) {
+            final String username = matcher.group("username");
+            res =
+                new AsyncResponse(
+                    this.settings.credentials().thenApply(
+                        cred -> cred.map(
+                            yaml -> Optional.ofNullable(yaml.yamlMapping(username))
+                            .<Response>map(
+                                ignored ->
+                                    new RsWithBody(
+                                        new RsWithStatus(RsStatus.OK),
+                                        ByteBuffer.wrap(
+                                            Json.createObjectBuilder()
+                                                .add("name", username)
+                                                .add(
```

Comment:
@olenagerasimova let's put required field such as `lastLoggedIn` and `realm` with stub values